tests/t220_trace_script.py

```diff
@@ -34,3 +34,8 @@ def __init__(self):
     def setup(self):
         self.option = "--force -F fork -F main"
         self.exearg = TEST_SCRIPT
+
+    def fixup(self, cflags, result):
+        result = result.replace(' 133.697 us [28137] | fork();\n', '')
+        result = result.replace('            [28141] | } /* fork */\n', '')
+        return result
```

Comment:
Can you share the output when it fails?  I guess we can use `task` sort function to keep the ordering for each task.

Reply:
Here is the diff. I also updated the commit message with this diff.
```diff
$ ./runtest.py -vdp -O0 -c gcc 220
      ...
t220_trace_script: diff result of -pg -O0
--- expect      2023-02-15 20:35:23.756503882 +0900
+++ result      2023-02-15 20:35:23.756503882 +0900
@@ -1,3 +1 @@
- fork();
- } /* fork */
  main() {

220 trace_script        : NG
```